src/style_context.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2013-2020, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use glib::object::IsA;
+use glib::translate::*;
+use gtk_sys;
+use libc::c_void;
+use pango::FontDescription;
+use pango_sys::PangoFontDescription;
+use StateFlags;
+use StyleContext;
+
+pub trait StyleContextExtManual: 'static {
+    fn get_font(&self, state: StateFlags) -> Option<FontDescription>;
+}
+
+impl<O: IsA<StyleContext>> StyleContextExtManual for O {
+    fn get_font(&self, state: StateFlags) -> Option<FontDescription> {
+        unsafe {
+            let font_description_ptr: *mut PangoFontDescription = std::ptr::null_mut();
+            gtk_sys::gtk_style_context_get(
```

Comment:
You can do this completely without unsafe code now by calling `get_property()` :)

Reply:
You right now I can 😉